nntrainer/tensor/cl_operations/blas_kernels_fp16.cpp

```diff
@@ -176,7 +166,8 @@ __fp16 dot_cl(const __fp16 *vecAdata, const __fp16 *vecXdata,
       break;
     }
 
-    result = dotResult.ReadData(cl_context_ref.command_queue_inst_, &cl_ret);
+    result = clbuffInstance.getOutBufferA()->ReadDataRegion(
+      cl_context_ref.command_queue_inst_, sizeof(cl_half), &cl_ret);
```

Comment:
Isn't it better to match the type btw/ `sizeof(cl_half)` and `cl_ret`?
```suggestion
      cl_context_ref.command_queue_inst_, sizeof(__fp16), &cl_ret);
```
where
https://github.com/nnstreamer/nntrainer/blob/252b2df9ae469505673c8dca1649214ec94ca046/nntrainer/tensor/cl_operations/blas_kernels_fp16.cpp#L116
Is there any particular reason you designate it to `cl_half`, which is defined as :
https://github.com/nnstreamer/nntrainer/blob/252b2df9ae469505673c8dca1649214ec94ca046/nntrainer/opencl/third_party/cl_platform.h#L142

Reply:
Thanks! I have updated it now.